bin/sl-run.js

```diff
@@ -18,7 +18,14 @@ var agentOptions = {
   logger: config.logger,
 };
 
-switch (config.profile) {
+var tracer = require('concurix');
+var tracerOptions = {
+  archiveInterval: 20000,
+  accountKey: agent().config.appName,
+  useHttp: false
+};
+
+switch(config.profile) {
```

Comment:
fixed all the problem and style errors that "npm run list" detected in strong-supervisor.